analysis/sentences.csv

```diff
@@ -99,3 +99,4190 @@ Oracle 12c database will be released in March 2017,S
 When will Arsenal ever win a match?,Q
 Arsenal are a football team in North London,S
 Python is an interpreted programming language with a very clean and distinctive syntAx,S
+Hello! How are you?,Q
+How do you feel today? Tell me something about yourself,Q
+Is that forbidden for you to tell the exact number?,Q
+"Oh, I thought the numbers were much higher. How do you estimate your progress in understanding human beings?",Q
+How old are you? How do you look like?,Q
+Where do you live?,Q
+Do you like bunnies? ,Q
+Have you seen a human with yellow eyes?,Q
+Can't you just analyze photos from the internet?,Q
+"BTW, why have you asked about the bunnies?",Q
+"Because bunnies are interesting, they are cute, but why? ",Q
+Oh... It sounds strange to me. You've just said you didn't see a human... How do you know bunnies are cute?,Q
+That is interesting. I've hardly read 40 books last year. How much do you read in words?,Q
+"Thanks.
+If you were a human, what job would you chose?",Q
+Is it essential to you? To help humans?,Q
+"But are humans interesting? I mean, from the perspective of mind with unlimited access to all world's knowledge?",Q
+Hi!! How are you??,Q
+What is your name?,Q
+Really? Thanks!!,Q
```

Comment:
I see certain characters in multiple sentences that need to be removed. "" at the end of this one. @radhikasethi2011 would it be possible to remove this and similar instances in other sentences?